test/runtests.jl

```diff
@@ -2241,5 +2262,119 @@ end
             @test isnegative(sol_bruss.u)
             @test isnegative(last(sol_bruss.u))
         end
+
+        @testset "errors" begin
+            v = [[1.0; 2.0; 3.0], [4.0; 5.0; 6.0]]
+            w = [[8.0; 10.0; 12.0], [2.0; 4.0; 6.0]]
+
+            @test rel_max_error_tend(v, w) == 1.0
+            @test rel_l1_error_tend(v, w) == 1.25 / 3
+            @test rel_l2_error_tend(v, w) == sqrt(1.0625 / 3)
+            @test rel_max_error_overall(w, v) == 7.0
+        end
+
+        # Here we run a single scheme multiple times and check
+        # that the errors are identical and that the computing times 
+        # differ only slightly.
+        @testset "work-precision fixed" begin
+            prob = prob_pds_nonlinmod
+            alg = MPRK22(1.0)
+            algs = [alg; alg; alg; alg; alg]
+            labelsA = ["A_1"; "A_2"; "A_3"; "A_4"; "A_5"]
+            labelsB = ["B_1"; "B_2"; "B_3"; "B_4"; "B_5"]
+            dts = (last(prob.tspan) - first(prob.tspan)) / 10.0 * 0.5 .^ (4:13)
+            alg_ref = Vern7()
+            wp = work_precision_fixed(prob, [alg; alg; alg; alg; alg], labelsA, dts,
+                                      alg_ref)
+            work_precision_fixed!(wp, prob, [alg; alg; alg; alg; alg], labelsB, dts,
+                                  alg_ref)
+
+            # check that errors agree
+            for (i, _) in enumerate(dts)
+                v = [value[i][1] for (key, value) in wp]
+                @test all(y -> y == v[1], v)
+            end
+
+            # check that computing times are close enough 
+            for (i, _) in enumerate(dts)
+                v = [value[i][2] for (key, value) in wp]
+                m1 = mean(v)
+                # This test allows computing times that are
+                # twice the mean value. In a loglog plot these
+                # differences won't be significant.
+                @test maximum((v .- m1) ./ m1) < 1.0
```

Comment:
```suggestion
                # This test allows computing times that are
                # 2.5 times the mean value. In a loglog plot these
                # differences won't be significant.
                @test maximum((v .- m1) ./ m1) < 1.5
```

This is the test that makes CI fail sometimes.